### ISSUE TITLE
[SI-REVIEW-TRUE] 1.1.1 This image has an empty alt text

### ISSUE THREAD

**Issue by kreynen** [CuBoulder/express#925]
Siteimprove is reporting that http://www.colorado.edu/sites/all/themes/custom/homepage2016/logo.png has no alt text on http://www.colorado.edu/

`<div id="sticky-menu" class="clearfix has-shadow"><a href="http://www.colorado.edu"><img src="http://www.colorado.edu/sites/all/themes/custom/homepage2016/logo.png" alt=" " />`
